### ISSUE TITLE
Looks Outdated 

### ISSUE THREAD

**Issue by fboulegue** [DrewDahlman/ImageFilter#2]
Plugin looks outdated for Cordova2.3

**Comment by DrewDahlman** [comment]
Yes it is. The upgrade shouldn't be too big of an issue due to the fact that most of the native side code is VERY native. Basically switching the imports of the Cordova framework. I have been meaning to get around to the upgrade but also very busy...

**Comment by fboulegue** [comment]
So there wont be a update from your side? 

Am Jan 25, 2013 um 5:05 PM schrieb Drew Dahlman notifications@github.com:

**Comment by ykarhus** [comment]
I'm having problems using it. Some variables at Filters.java dont exist anywhere else.

**Comment by ykarhus** [comment]
I'm trying to save an image after it pass by a filter. I get a canvas base64 string and with this I try to save a PNG/JPEG image but your plugin seems to do the filter and save parts without trouble. Can you help me?

**Comment by DrewDahlman** [comment]
what variable are missing? Make sure you're updating your app.java to open up the context...

**Comment by ykarhus** [comment]
It doesn't recognizes the R variable and the Context class. Does it really applies a filter on the image and make it possible to download the image on the android??

**Comment by DrewDahlman** [comment]
Okay you need to adjust your app.java class and open up the context. https://github.com/DrewDahlman/ImageFilter/blob/master/Android/project/src/com/drewdahlman/ImageFilter/App.java there's an example of what it should look like.

Also yes - this plugin will apply a filter to your image and save it out- it will return a url path to that image to your js which you can upload to a server or save locally to your phone.

**Comment by ykarhus** [comment]
Thank you. I'll try to do it!!

**Comment by ykarhus** [comment]
Will be any problem if I use Cordova 2.5.0 ?

**Comment by DrewDahlman** [comment]
Unfortunately yes… the plugin was made for phonegap before the cordova change…

There needs to be updates made - shouldn't be too much but I haven't had the time to make this update recently…

**Comment by ykarhus** [comment]
And I think i dont get the competence to do it. I'll try to use it with the version you used and then I'll try myself to make the upgrade.

**Comment by ykarhus** [comment]
I made it to run with the version of phonegap you used but the image desappears suddenly right after it gets in the square in the middle of the app!!! o.O ??

**Comment by ykarhus** [comment]
And when I apply the filter it still doesnt appears and I'm not able to save it either.
